tool/speed.cc

```diff
@@ -357,11 +357,19 @@ static bool SpeedRSAKeyGen(const std::string &selected) {
       BM_NAMESPACE::UniquePtr<RSA> rsa(RSA_new());
 
       const uint64_t iteration_start = time_now();
+#if defined(AWSLC_FIPS)
+      if (!RSA_generate_key_fips(rsa.get(), size, nullptr)) {
+        fprintf(stderr, "RSA_generate_key_fips failed.\n");
+        ERR_print_errors_fp(stderr);
+        return false;
+      }
+#else
       if (!RSA_generate_key_ex(rsa.get(), size, e.get(), nullptr)) {
         fprintf(stderr, "RSA_generate_key_ex failed.\n");
         ERR_print_errors_fp(stderr);
         return false;
       }
+#endif
```

Comment:
Question for others: is it worth just running both all the time? And reporting a RSA keygen and a RSA FIPS keygen time?

Reply:
Added support for both :)